### DISCUSSION TITLE
Can we use "sequencer" field in bucket notification to determine the sequence of events in a MinIO cluster

### DISCUSSION THREAD

**Discussion by fangzhouxie** [minio/minio#18220] [Q&A] [answered]
Hi team, I have a few questions about the sequencer field in bucket notifications. I see in [MinIO bucket notifications documentation](https://min.io/docs/minio/linux/administration/monitoring/bucket-notifications.html#bucket-notifications) that it is similar to S3 event notifications, and in [S3 event message structure](https://docs.aws.amazon.com/AmazonS3/latest/userguide/notification-content-structure.html) that "The sequencer key provides a way to determine the sequence of events.". I see in the code that sequencer field in bucket notification is generated based on current timestamp. I'm wondering can we use it to determine the sequence of events in a MinIO cluster?

**Comment by klauspost** [comment, marked as answer]
There is no sequence synchronization, so no.

**Reply by fangzhouxie** [reply]
Thank you for the clarification!

Another quick follow up question - can we use "sequencer" field to determine order of events for _the same object_? My understanding is no but wanted to double check.
